app/src/main/assets/drug_stock_no_connection.html

```diff
@@ -0,0 +1,8 @@
+<html>
```

Comment:
Can we format this HTML, please?

Reply:
I did, Android Studio and Idea is formatting like this 🤷🏾

Reply:
I will manually format the file and push the changes.